lib/meta-boxes.php

```diff
@@ -236,5 +236,59 @@ function igv_cmb_metaboxes() {
     ),
   ) );
 
+
+  // ATTACHMENTS
+
+  $attachment_metabox = new_cmb2_box( array(
```

Comment:
These dont seem to show up anywhere I can find in my local though the codebase looks correct. Where should I be looking?

Reply:
ooops sorry. should have xplained.

Media > Click an attachment > Scroll down > Edit more details